src/ensRegistry.ts

```diff
@@ -59,12 +60,11 @@ export function handleNewOwner(event: NewOwnerEvent): void {
 // Handler for Transfer events
 export function handleTransfer(event: TransferEvent): void {
   let node = event.params.node.toHexString()
-
   let account = new Account(event.params.owner.toHexString())
   account.save()
 
   // Update the domain owner
-  let domain = new Domain(node)
```

Comment:
This isn't necessary here - creating a new one and writing it will merge it with the existing record if there is one.